### ISSUE TITLE
Error Downloading OpenAPI Spec

### ISSUE THREAD

**Issue by kayx23** [Redocly/redoc#2318]
For example with Instagram's: 

https://redocly.github.io/redoc/?url=https://api.apis.guru/v2/specs/instagram.com/1.0.0/swagger.yaml

![image](https://github.com/Redocly/redoc/assets/39619599/8401bb04-e702-4ee8-917c-8e484354c722)

I get 
```
Missing required request header. Must specify one of: origin,x-requested-with
```

Observed the same thing with other openAPI specs downloads.

**Comment by romanpryshliak** [comment]
Hi @kayx23,

Please attempt to download the spec with CORS disabled:

https://redocly.github.io/redoc/?url=https://api.apis.guru/v2/specs/instagram.com/1.0.0/swagger.yaml&nocors

**Comment by kayx23** [comment]
@romanpryshliak Yep that worked. However if I recall correctly, this worked some time ago without disabling CORS?

Currently the download doesn't work on either Safari or Chrome. Is this the intended behaviour?
